images/base-notebook/docker_healthcheck.py

```diff
@@ -3,14 +3,24 @@
 # Distributed under the terms of the Modified BSD License.
 import json
 import os
+import warnings
 from pathlib import Path
 
 import requests
 
+# Import Jupyter paths ignoring deprecation warning that does not affect this script
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    from jupyter_core import paths
+
+# Set HOME according to the user in NB_USER
+# This is necessary for paths.jupyter_runtime_dir() to work correctly
+os.environ["HOME"] = str(Path("/home") / os.environ["NB_USER"])
+
 # Several operations below deliberately don't check for possible errors
 # As this is a healthcheck, it should succeed or raise an exception on error
```

Comment:
Let's change these 2 sections

Reply:
Apologies, the Github review history became a bit confusing to read for me now :)
What change exactly did you mean?

Reply:
I prefer keeping "Several operations below deliberately don't check..." at the top of this file, because it describes the script design.

Reply:
Oh you meant re-ordering them. Sure!

Reply:
Sorry, I meant swap. It's been a long day 😆